_transformations/scale/readme.md

```diff
@@ -14,17 +14,43 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges in a scale file.
```

Comment:
I feel the sentence would be easier to understand without the ending "in a scale file"